front-end/src/ui-components/CouncilSignalBar.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import styled from '@xstyled/styled-components';
+import React from 'react';
+
+interface Props {
+	className?: string
+	ayeSignals: number
+	naySignals: number
+	councilSize?: number
+}
+
+const CouncilSignalBar = ({ ayeSignals, className, councilSize = 13, naySignals }: Props) => {
```

Comment:
This = 13 doesn't sound future proof :)

Reply:
That's on me (from the first component mock) :) - but yeah, `councilSize` should come from the api